File: tools/dbench/answers/go_haiku/s1/m1_oldest.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	file, _ := os.Open("users.txt")
	defer file.Close()

	scanner := bufio.NewScanner(file)
	var oldestName string
	maxAge := -1

	for scanner.Scan() {
		fields := strings.Fields(scanner.Text())
		if len(fields) != 3 {
			continue
		}
		age, err := strconv.Atoi(fields[2])
		if err != nil {
			continue
		}
		if age > maxAge {
			maxAge = age
			oldestName = fields[1]
		}
	}

	fmt.Printf("%s %d\n", oldestName, maxAge)
}

File: tools/dbench/answers/python_haiku/s3/m1_oldest.py
l = open("users.txt").readlines()
n = None
a = -1
for x in l:
    p = x.strip().split()
    if len(p) == 3:
        try:
            g = int(p[2])
            if g > a:
                a = g
                n = p[1]
        except:
            pass
if n:
    print(f"{n} {a}")
